blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserImpl.java

```diff
@@ -36,15 +47,17 @@
 
     protected final User user;
 
+    private AccessControlled orgBase;
+
     private final Reachable parent;
     public UserImpl(User user, Reachable parent) {
```

Comment:
ummmm, I thought about adding that parameter to UserImpl but then I missed the container and went back... changing it.